src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,35 @@
+.icon {
+  display: block;
+  height: 24px;
+  width: 24px;
+  background-size: cover;
+  position: relative;
+
+  @include hover(transform, scale(1.1));
+
+  @include on-tablet {
+    height: 32px;
+    width: 32px;
+  }
+
+  &--phone {
+    background-image: url(/src/images/icons/Icon-phone-call.svg);
+  }
+
+  &--phone:hover + .header__contacts,
+  &--phone:focus + .header__contacts {
```

Comment:
i can click on that area and it will work like a button. Fix it

<img width="1026" alt="Знімок екрана 2024-11-12 о 10 55 06" src="https://github.com/user-attachments/assets/7d94f550-9bab-4813-a15d-bf4c15f5afee">